package.json

```diff
@@ -18,8 +18,8 @@
     "@nextui-org/theme": "^2.2.4",
     "@nextui-org/tooltip": "^2.0.34",
     "@reduxjs/toolkit": "^1.8.1",
-    "@secured-finance/sf-client": "^0.2.0-beta.176",
-    "@secured-finance/sf-core": "0.2.0-beta.152",
+    "@secured-finance/sf-client": "^0.2.0-beta.177",
+    "@secured-finance/sf-core": "^0.2.0-beta.177",
```

Comment:
These libraries must be configured with the fixed versions.
So please use `--save-exact` option when npm install.

Reply:
updated